### PR TITLE
Create project settings page

### PR DESCRIPTION
**Pull request by DaniilPoiarkov**
Added ability to customize issue template with it's Label fields and Dropdown Fields
Routes: 
--project/settings - just buttons to navigate
--project/issue-template - change issue template

### CODE REVIEW

File: frontend/src/app/components/tasque-project-settings/project-options-module/issue-template/dropdown-field-edit/dropdown-field-edit.component.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    </div>
+    <div class="create-dropdown">
+        <span>Add new option</span>
+        <tasque-button style="margin-left: 10px" class="btn icon" [icon]="faPlus" (btnClick)="addField()"></tasque-button>
+    </div>
+    <tasque-button style="margin-left: 10px" class="btn mini" text="Finish editing" (btnClick)="finishEdit()"></tasque-button>
+    <div class="data-save">
+        <span>Data will be saved</span>
+    </div>
+</div>
```
Comment (FeloniukM) [inline frontend/src/app/components/tasque-project-settings/project-options-module/issue-template/dropdown-field-edit/dropdown-field-edit.component.html]: Forgot a space. Same in a few more files as well

Comment (DaniilPoiarkov) [reply frontend/src/app/components/tasque-project-settings/project-options-module/issue-template/dropdown-field-edit/dropdown-field-edit.component.html]: Already fixed

**Review by FeloniukM — DISMISSED** [review]

File: backend/Tasque.Core.Common/Entities/Task/DropdownField.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Tasque.Core.Common.Entities.Task
+{
+    public class DropdownField
+    {
+        public string[] Fields { get; set; }
```
Comment (Anatolii-Ivanov) [inline backend/Tasque.Core.Common/Entities/Task/DropdownField.cs]: Not for this PR, but can we make all fields more generic?

File: frontend/src/app/components/tasque-project-settings/project-options-module/issue-template/issue-template.component.ts [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  public contextFields: TaskCustomField[] = [ 
+    { name:'Assignee', type: TaskFieldType.User }, 
+    { name:'Label', type: TaskFieldType.Label, labels: [ 
+      { color: 'red', name: 'first label' },
+      { color: 'green', name: 'second label' },
+      { color: 'blue', name: 'third label' }, 
+    ] }, 
+    { name:'Sprint', type: TaskFieldType.Type }, 
+    { name:'Story point estimate', type: TaskFieldType.Number }
+  ];
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/components/tasque-project-settings/project-options-module/issue-template/issue-template.component.ts:42]: I hope you will remove this custom data

Comment (DaniilPoiarkov) [reply frontend/src/app/components/tasque-project-settings/project-options-module/issue-template/issue-template.component.ts:42]: Yes, this fields will be taken from existing task templates by fetching data. This was only for test how implemented functionality works. At this time I need to use this data with functionality in branch with noSql to find a better way to connect both features together.

File: backend/Tasque.Core.Common/Entities/Task/DropdownField.cs (hunk 2 above)
Comment (DaniilPoiarkov) [reply backend/Tasque.Core.Common/Entities/Task/DropdownField.cs]: Removed from this PR. Yes and I guess I know how, but I need to use this data in branch with cosmosDb functionality

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by FeloniukM — APPROVED** [review]

**Pull request merged**